packages/mask/src/plugins/RedPacket/SNSAdaptor/index.tsx

```diff
@@ -94,7 +94,7 @@ const sns: Plugin.SNSAdaptor.Definition = {
     ApplicationEntries: [
         (() => {
             const icon = <RedPacketIcon />
-            const name = <Trans i18nKey="plugin_red_packet_name" />
+            const name = <Trans i18nKey="com.maskbook.red_packet:name" />
```

Comment:
Wait... You should use 

```js
import { useI18N } from '../locales'
const { t } = useI18N()
t.name()
```

Reply:
Not all `<Trans />` here are in React component directly. Using `<Trans />` directly is more convenient.